src/sync/sensor/processors/sensor.processor.ts

```diff
@@ -11,15 +11,55 @@ import {
   SensorRelocated,
   SensorUpdated,
 } from 'src/events/sensor';
+import {LedgerConnection} from "../ledger.connection";
+const { Gateway, Wallets } = require('fabric-network');
+const path = require('path');
+const fs = require('fs');
 
 @Injectable()
 export class SensorProcessor {
   protected logger: Logger = new Logger(this.constructor.name);
 
+  constructor(private readonly ledgerConnection: LedgerConnection) {
+  }
+
   async process(event): Promise<void> {
```

Comment:
Dit zou in een `AbstractSensorProcessor` class kunnen, niet? Die ondersteunt alle events standaard met een 'Logger iets' of zo? Of alleen abstract functions?